src/app/shared/shared.module.ts

```diff
@@ -28,8 +28,7 @@ import { MaterialModule } from './material.module';
   imports: [
     CommonModule,
     IconsModule,
-    MaterialModule,
-    ReactiveFormsModule
+    MaterialModule
```

Comment:
Please revert all the changes made in this file.

Reply:
Material module and reactiveforms module?

Reply:
Adjusted as in the master branch shared.module.ts